server/src/main/kotlin/suwayomi/tachidesk/manga/impl/download/DownloadManager.kt

```diff
@@ -47,6 +54,38 @@ object DownloadManager {
     private val downloadQueue = CopyOnWriteArrayList<DownloadChapter>()
     private val downloaders = ConcurrentHashMap<Long, Downloader>()
 
+    private const val downloadQueueKey = "downloadQueueKey"
+    private val sharedPreferences =
+        Injekt.get<Application>().getSharedPreferences(DownloadManager::class.jvmName, Context.MODE_PRIVATE)
+
+    private fun loadDownloadQueue(): List<Int> {
+        return try {
+            sharedPreferences.getStringSet(downloadQueueKey, emptySet())?.map { it.toInt() } ?: emptyList()
```

Comment:
```suggestion
            sharedPreferences.getStringSet(downloadQueueKey, emptySet())?.mapNotNull { it.toIntOrNull() } ?: emptyList()
```
Just to be safe.
Also I don't think you need the try catch statements anymore

Reply:
oh yeah, forgot to remove them